### PR TITLE
Display Conversion USD > CDF

### PR DESCRIPTION
**Pull request by MaganaAsiati**
Rationale

Display the amount of the SO in USD

Specification

If the currency of the pricelist used on the SO is different from the currency of the company,

display the amount in the currency of the company 

image

The total in USD is computed when the total of the SO is changed. 

Once the SO is confirmed, the rate used is the one on the day of the confirmation.

If the SO is edited afterwards (cancel > back to quotation), recompute the total.

On the SO printed, display the amount on USD and the rate below the total price

image

Same on the invoices

### CODE REVIEW

**Comment by iambenkis** [comment]
Hi @MaganaAsiati,

Good job working on this milestone, However, while looking closely at your code I've noticed that you have unrelated module that are not yet merged because this branch is not created from the main branch. 

Kindly make sure you do it before I proceed with the review.

Keep doing the great work 👍

File: gse_Conversion/views/report_templates.xml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                <tr class="border-black o_total">
+                    <td><strong>Amount in Company Currency</strong></td>
+                    <td class="text-end">
+                            <span t-field="doc.company_currency_amount"/>
+                    </td>
+                </tr>
+             </t>
+            </xpath>
+        </template>
+        <template id="report_saleorder_document_copy_assia" inherit_id="gse_custo.report_saleorder_document_copy">
```
Comment (sbuhl) [inline gse_Conversion/views/report_templates.xml:23]: @MaganaAsiati 
tu fais appel à gse_custo, mais il n'est pas dans les dépendances...

File: gse_Conversion/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        Customisation of Field Service module for GoShop Energy""",
+    "description": """
+    """,
+    "author": "Magana Mwinja Asiati",
+    "website": "https://github.com/GoShop-Energy/field-service",
+    "category": "Customizations",
+    "version": "0.1.8.7",
+    "license": "LGPL-3",
+    "depends": [
+        "base",
```
Comment (sbuhl) [inline gse_Conversion/__manifest__.py:15]: @MaganaAsiati 
Are all those needed?

**Comment by sbuhl** [comment]
@MaganaAsiati  if you created a new PR, can you close this one please.

**Pull request closed without merging**